examples/step-79/doc/intro.dox

```diff
@@ -160,21 +160,21 @@ Typically, the solutions to topology optimization problems are not
 mesh-independent, and as such the problem is ill-posed. This is because as the
 mesh is refined further, fractal structures are often formed. As the mesh gains
 resolution, the optimal solution typically gains smaller and smaller structures.
-There are a few competing work-arounds to this issue, but the most popular for
+There are a few competing workarounds to this issue, but the most popular for
 first order optimization is the sensitivity filter, while second order
 optimization methods tend to prefer use of a density filter.
 
 As the filters affect the gradient and Hessian of the compliance (i.e., the
 objective function), the choice of filter has an effect on the solution of the
 problem. The density filter as part of a second order method works by
-introducing an unfiltered density, which we refer to as $\sigma$, and then
+introducing an unfiltered density, which we refer to as $\varrho$, and then
```

Comment:
Here I swapped the `\sigma` symbol used for the unfiltered density with `\varrho`. If you're not happy with this choice then we should pick another one, but `\sigma` is too easily confused with the stress so I'd strongly encourage you to make *a* change here. Its definitely necessary to check that I got the changes right (i.e. where changes were made, with `\sigma` actually represent the unfiltered density, or was it in fact the stress? I think that you used the constitutive law to prevent the direct need for the stress tensor from this point on, but I could be wrong about that...